### DISCUSSION TITLE
OpenSSL 0.9.8 on Vulnerability check gives critical threat warning.

### DISCUSSION THREAD

**Discussion by prmjh4** [openssl/openssl#25103] [Q&A]
I ran the vulnerability scan check on my application that used OpenSSL 0.9.8 and it throws it in the critical section threat to the application.
Description as given by the Scan:
The version of OpenSSL installed on the remote host is prior to 0.9.8m. It is, therefore, affected by multiple vulnerabilities as referenced in the 0.9.8m advisory.
Solution as given by the Scan:
Upgrade to OpenSSL version 0.9.8m or later.
Path : /opt/lib/extra/libcrypto.so.0.9.8
Reported version : 0.9.8i
Fixed version : 0.9.8m
Path : /opt/lib/extra/libssl.so.0.9.8
Reported version : 0.9.8i
Fixed version : 0.9.8m
I couldn't find the OpenSSL0.9.8m version anywhere so I instead downloaded the OpenSSL 1.1.1w but then it instead started displaying the below error message under critical vulnerability.
 Path             : /opt/lib/extra/libcrypto.so.1.1
  Reported version : 1.1.1w
  Fixed version    : 1.1.1za
I couldn't found the OpenSSL1.1.1za version anywhere so what can I do now to resolve the error.
I have a constraint that I have to stick with the OpenSSL 1.x series version because of many dependency issues, how can I makes sure that issue gets removed from critical section. Please help me out with it!

**Comment by tom-cosgrove-arm** [comment]
> I have a constraint that I have to stick with the OpenSSL 1.x series version because of many dependency issues

OpenSSL 1.1.1 was released in 2018 as an LTS release, to be supported for 5 years, and consequently became end-of-life on 11th September 2023. The EOL date was reconfirmed in March 2023. If you absolutely must have support for it, premium support contracts are available from the OpenSSL Corporation - see https://openssl-corporation.org/support/.
